portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -104,7 +104,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     UserService userService = UserServiceFactory.getUserService();
     
     // Get the user, comment, and image from the form and add it to the array.
```

Comment:
Consider updating this comment to reflect your latest code changes!

Reply:
Done!